utils/rpms/daos.spec

```diff
@@ -386,6 +386,9 @@ getent passwd daos_server >/dev/null || useradd -M daos_server
 %{_libdir}/*.a
 
 %changelog
+* Thu Oct 1 2020 Alexander Oganezov <alexander.a.oganezov@intel.com> 1.1.0-34
+- Set OFI to compile with --disable-efa flag to disable EFA provider
```

Comment:
don't think spec change is necessary here since the change doesn't affect the RPM build

Reply:
Agreed.